dev/io.openliberty.org.apache.myfaces.4.0_fat/bnd.bnd

```diff
@@ -44,4 +45,4 @@ javac.target: 11
 	io.openliberty.jakarta.servlet.6.0;version=latest,\
 	io.openliberty.jakarta.annotation.2.1;version=latest,\
 	io.openliberty.jakarta.expressionLanguage.5.0;version=latest,\
-
+	jakarta.enterprise.cdi-api
```

Comment:
Does this need a version? Should this be a bundle / project instead?

Reply:
Oops.  Not even sure how that got in there.  Probably a mistake in resolving merge conflicts.